src/hooks/settingsHooks/useGetSettings.ts

```diff
@@ -127,17 +127,25 @@ const extractPassword = (
 }
 
 export const useGetSettings = (
-  siteName: string
+  siteName: string,
+  isEmailLogin?: boolean
 ): UseQueryResult<SiteSettings> => {
+  const shouldGetPrivacyDetails =
+    isEmailLogin === undefined ? false : isEmailLogin
   return useQuery<SiteSettings>(
-    [SETTINGS_CONTENT_KEY, siteName],
+    [SETTINGS_CONTENT_KEY, siteName, shouldGetPrivacyDetails],
     async () => {
       const siteSettings = await SettingsService.get({ siteName })
-      // TODO: reimplement actual functionality once netlify issue resolved
-      // const passwordSettings = await SettingsService.getPassword({ siteName })
-      const passwordSettings = {
-        isAmplifySite: false,
-        password: "",
+      let passwordSettings
+      const isLaunchDarklyImplemented = false
+      if (shouldGetPrivacyDetails && isLaunchDarklyImplemented) {
+        // TODO: LaunchDarkly to allow specific groups to access this feature first
+        passwordSettings = await SettingsService.getPassword({ siteName })
+      } else {
+        passwordSettings = {
+          isAmplifySite: false,
+          password: "",
+        }
```

Comment:
why's this here ._. also, we should remove the toggle as a whole rather than simply changing the request (+ enforce on BE also)

we should also use an env var for this because having it in code requires a full deploy, which defeats the purpose

Reply:
Accidental merge into the wrong branch - will put this in the main branch!

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/1316/commits/b26ab91a104f550adf0b34beb59e0f6d1654de41